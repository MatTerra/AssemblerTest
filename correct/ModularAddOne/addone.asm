BEGIN
test: extern
PUblic start
 section data
one:        CONST 1
two:        CONST -2


            section text
start:      add     one                 ; Simple add something
            jmp     finishing
store test
copy one,test
; Let's finish this empty file
finishing:        stop
     
END
