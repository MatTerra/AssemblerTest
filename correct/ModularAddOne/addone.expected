PC <- 2
ACC <- 1

PC <- 9
ACC <- 1

PC <- 10
ACC <- 1

